### ISSUE TITLE
How to handle node where aws kinesis input thread is running

### ISSUE THREAD

**Issue by umeshdc** [Graylog2/graylog-plugin-aws#72]
Hi,

We have 2 node graylog cluster and we found that at any point aws kinesis input thread will be running in only one graylog node. 

Also we found that if graylog node where this thread is running goes down, their no way that active graylog will recreate that thread to continue consuming messages.

Please let us know if their is a way to handle this scenario.

Thanks

**Comment by joschi** [comment]
@umeshdc Have you tried starting the input as a global input (which is running on all nodes)?

**Comment by umeshdc** [comment]
Yes, If we choose global option while creating input, then thread will be created in randomly selected node. If we select specific node then thread will be always created in the selected node.

**Comment by gabricar** [comment]
I have the same issue. Global option does not work
